### DISCUSSION TITLE
--log-level doesn't seem to do anything

### DISCUSSION THREAD

**Discussion by aakoch** [parcel-bundler/parcel#6787] [Q&A]
Am I calling it incorrect? I see no more messages printed than when I don't have it. I've seen errors when I try a log level doesn't exist, so I _think_ I'm doing it correct...
`yarn parcel src/index.pug src/**/index.pug --log-level verbose`

`npx parcel build --no-cache --log-level verbose ./src/index.pug './src/**/index.pug'`

Any guidance would be appreciated.

Thanks,
Adam

**Comment by mischnic** [comment]
The usage is correct, there just aren't that many additional log points at the verbose level. Were you expecting some specific messages here?

**Reply by aakoch** [reply]
Hmmm... I was hoping to see more stuff like when the image transformer was running and on which files, etc.  I just thought it was weird I am not see any more log messages.
